.eslintrc

```diff
@@ -48,14 +61,30 @@
           "src/core/polyfills.js"
         ]
       }
-    ]
+    ],
   },
   "overrides": [
     {
-      "files": ["**/*.dev.jsx"],
+      "files": [
+        "*.js",
+        "*.jsx"
+      ],
       "rules": {
-        "react/jsx-props-no-spreading": "off"
+        // These rules were triggered on the former non-typescript codebase.
+        // We are planning to use only ts/tsx in the future
+        // Therefor we can seperate them by only being ignored on js/jsx files.
+        // Start - ddb-react former code
+        "react/jsx-no-bind": "off",
+        "react/jsx-props-no-spreading": "off",
+        "react/function-component-definition": "off",
+        "react/forbid-prop-types": "off",
+        "react/destructuring-assignment": "off",
+        "@typescript-eslint/comma-dangle": "off",
+        "@typescript-eslint/return-await": "off",
+        "no-param-reassign": "off",
+        "@typescript-eslint/no-var-requires": "off"
+        // End - ddb-react former code
       }
-    }
+    },
   ]
-}
+}
```

Comment:
Git usually likes a newline at the end of files.

I expect it should be possible to automate this in VSCode. You can also [ensure newlines are inserted it with `.editorconfig`](https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support).

Reply:
Yes. I had some weird setting in vscode that kept on deleting last newline.
I will add .editorconfig and and this extension enforces the rules:
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig